server.py

```diff
@@ -29,6 +30,26 @@ def list():
 			cache_timeout=0)
 
 
+@app.route("/geoip")
+def geoip():
+	ip = request.remote_addr
+
+	if ip.startswith("::ffff:"):
+		ip = ip[7:]
+	try:
+		geo = reader.get(ip)
+	except geoip2.errors.GeoIP2Error:
+		app.logger.warning("GeoIP lookup failure for %s." % (ip,))
+		return "unknown", 500
+
+	if geo and "continent" in geo:
+		return geo["continent"]["code"], 200
```

Comment:
For future extensibility I'd prefer returning JSON here.

Reply:
Sure, that's completely trivial anyway. Would you want to have a simple key-value structure? Like this:
```
{
    "continent": "NA"
}
```